addons/crm/views/crm_lead_views.xml

```diff
@@ -576,47 +576,114 @@
             </field>
         </record>
 
+        <act_window name="Send email"
+                res_model="mail.compose.message"
+                binding_model="crm.lead"
+                binding_views="list"
+                view_mode="form"
+                target="new"
+                id="crm_lead_act_window_compose"
+                context="{
+                            'default_composition_mode': 'comment',
+                            'default_email_to':'{($object.email or \'\')|safe}',
```

Comment:
Normally setting a default email_to is not necessary as in comment mode actually you post on the record + email_template_opportunity_mail already defines email_to / partner_to